### ISSUE TITLE
Unnecessary violation by Indentation check

### ISSUE THREAD

**Issue by SGanguly1999** [checkstyle/checkstyle#10015]
Detected in comment: [#9729](https://github.com/checkstyle/checkstyle/pull/9729#issuecomment-841773349)

I have read check documentation: https://checkstyle.sourceforge.io/config_misc.html#Indentation
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

```bash

/var/tmp $ javac TestClass.java 

/var/tmp $ cat config.xml

<?xml version="1.0"?>
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
        "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
<module name="Checker">
    <module name="TreeWalker">
        <module name="Indentation">
        </module>
    </module>
</module>

/var/tmp $ cat TestClass.java

public class TestClass {
    void
        method
            (
            )
    {
    }
}
/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-8.42-all.jar -c config.xml TestClass.java

Starting audit...
[ERROR] /~/TestClass.java:4:13: 'method def lparen' has incorrect indentation level 12, expected level should be 4. [Indentation]
[ERROR] /~/TestClass.java:5:13: 'method def rparen' has incorrect indentation level 12, expected level should be 4. [Indentation]
Audit done.
Checkstyle ends with 2 errors.
```

---

**EXPECTED**:

```
/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-8.42-all.jar -c config.xml TestClass.java

Starting audit...
Audit done.
```

---

Checkstyle shouldn't give violation in this case.

**Comment by Venkatesh-24** [comment]
I am on it

**Comment by Venkatesh-24** [comment]
@romani can you please point me to similar type of solved issues or guide me through to solve this issue

**Comment by romani** [comment]
Please find closed issues with label "indentation" and you will see related PRs/commits.
